docs/reference/modules/node.asciidoc

```diff
@@ -236,8 +248,9 @@ To create a dedicated content node, set:
 node.roles: [ data_content ]
 ----
 
+[role="xpack"]
 [[data-hot-node]]
-==== [x-pack]#Hot data node#
+==== Hot data node
 
 Hot data nodes store time series data as it enters {es}. The hot tier must be fast for
 both reads and writes, and requires more hardware resources (such as SSD drives).
```

Comment:
@debadair Same Q re: the allocation of data stream indices to the hot tier. We already mention it in the data tiers docs.